staging/src/k8s.io/pod-security-admission/api/helpers.go

```diff
@@ -144,12 +144,59 @@ func (lv LevelVersion) String() string {
 	return fmt.Sprintf("%s:%s", lv.Level, lv.Version)
 }
 
+// Determines whether two LevelVersions are functionally equivalent. LevelVersions are considered
+// equivalent if both are privileged, or both levels & versions are equal.
+func (lv *LevelVersion) Equals(other *LevelVersion) bool {
```

Comment:
I'd suggest using a more descriptive name like `Equivalent`. `Equals` is confusing if `lv != other`